src/Models/Requests/PostScriptResultV3.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Bank2Loyalty\Models\Requests;
+
+use Bank2Loyalty\Models\Data\Bank2LoyaltyInfo;
+use Bank2Loyalty\Models\Data\EmvInfo;
+use Bank2Loyalty\Models\Data\GoogleSmartTapInfo;
+use Bank2Loyalty\Models\Data\IdCardInfo;
+use Bank2Loyalty\Models\Data\ProgramScannedInfo;
+use Bank2Loyalty\Models\Data\ReaderInfo;
+use Bank2Loyalty\Models\Scripting\ScriptActionResults;
+
+class PostScriptResultV3 extends BaseRequestV3
```

Comment:
Only scriptActionResults should be needed? The rest is specified in BaseRequestV3?